test/docfx.Test/lib/YamlUtilityTest.cs

```diff
@@ -299,7 +299,7 @@ public void TestDuplicatedKeys()
             {
                 Assert.Equal(ErrorLevel.Error, error.Level);
                 Assert.Equal("yaml-syntax-error", error.Code);
-                Assert.Contains("Duplicate key", error.Message);
+                Assert.Contains("An item with the same key has already been added. Key: Key1", error.Message);
```

Comment:
>An item with the same key has already been added. Key: Key1 [](start = 33, length = 59)

I don't think this message is clear enough for users to take action. How about `Key 'Key1' is already defined. Please remove the duplicate key.`

Reply:
[Per MS style guide, avoid please](https://github.com/dotnet/docfx/commit/3ea6ddb104647ec432900f2361f6e1540c57b1ba)

---
In reply to: [200588479](https://github.com/dotnet/docfx/pull/3009#discussion_r200588479) [](ancestors = 200588479)